### PR TITLE
Fix Solana createOFTAdapter

### PR DESCRIPTION
**Pull request by DanL0**
make createOFTAdapter runnable again

tested using token2022 oft program (tried with mint authority and later renounced it):
```
{
    "programId": "3ZuKaovZkFLf5StLP2sHDBa65WKR7fjD1AViFugCFqRH",
    "mint": "E9n1uWWRzxZPKHq3fxzg1xaA1mDDBZ6dveNwRG4z6AY",
    "mintAuthority": "",
    "escrow": "5ktCX8rAjVpBTwxDWY2Uktm8EorkMKdQF6hAVNVi2nsF",
    "oftStore": "2beDM7DgWFutxXzC7TSmZh8rsP9b2mCyHyuutTJLSDnk"
}
```

successful tx: https://testnet.layerzeroscan.com/tx/usD6d3dN41h5yZNgNKLveR3PdgyuKxnLqfbbTMcBhLqYwhJgT2moNv7SUM2eczw3jMb3BXSkZbj7ZNuZHMMGPXG

### CODE REVIEW

File: examples/oft-solana/tasks/solana/createOFTAdapter.ts [hunk 1]
```diff
@@ -35,33 +48,57 @@ interface CreateOFTAdapterTaskArgs {
...
                 toWeb3JsPublicKey(oftStorePda),
-                toWeb3JsPublicKey(tokenProgram)
+                toWeb3JsPublicKey(tokenProgram),
+                additionalMinters
             )
+            console.log(`created SPL multisig @ ${mintAuthorityPublicKey.toBase58()}`)
+            await checkMultisigSigners(connection, mintAuthorityPublicKey, [
+                toWeb3JsPublicKey(oftStorePda),
+                ...additionalMinters,
+            ])
```
Comment (ryandgoulding) [inline examples/oft-solana/tasks/solana/createOFTAdapter.ts:96]: Why is this needed for OFTAdapter?  OFTAdapter is meant to be lock/unlock.  If folks want to modify their mint to support SPL multisig, that is outside of LayerZero scope.

Comment (DanL0) [reply examples/oft-solana/tasks/solana/createOFTAdapter.ts:96]: Oups, I thought it's needed but it seems it's only needed for logging stuff. Will remove. 😅

File: examples/oft-solana/tasks/solana/createOFTAdapter.ts [hunk 2]
```diff
@@ -49,19 +47,23 @@ task('lz:oft-adapter:solana:create', 'Mints new SPL Token, Lockbox, and new OFT
...
-            const mintAuthorityPublicKey = await createMintAuthorityMultisig(
-                connection,
-                toWeb3JsKeypair(umiWalletKeyPair),
-                toWeb3JsPublicKey(oftStorePda),
-                toWeb3JsPublicKey(tokenProgram)
-            )
+
+            let mintAuthority: PublicKey | null = null
+
+            if (tokenProgramStr.toLowerCase() === TOKEN_PROGRAM_ID.toBase58().toLowerCase()) {
```
[the remark this answers is not in the document]
Comment (DanL0) [reply examples/oft-solana/tasks/solana/createOFTAdapter.ts]: Fixed and tested.

File: examples/oft-solana/tasks/solana/createOFTAdapter.ts [hunk 3]
```diff
@@ -74,6 +76,6 @@ task('lz:oft-adapter:solana:create', 'Mints new SPL Token, Lockbox, and new OFT
             const { signature } = await transactionBuilder().add(initOftIx).sendAndConfirm(umi)
             console.log(`initOftTx: ${getExplorerTxLink(bs58.encode(signature), eid == EndpointId.SOLANA_V2_TESTNET)}`)
 
-            output(eid, programIdStr, mint, mintAuthorityPublicKey.toBase58(), escrowPK, oftStorePda)
+            output(eid, programIdStr, mint, mintAuthority ? mintAuthority.toBase58() : '', escrowPK, oftStorePda)
```
[the remark this answers is not in the document]
Comment (DanL0) [reply examples/oft-solana/tasks/solana/createOFTAdapter.ts:75]: We can keep it as it is. It's not a big deal. The field can be empty though if you pass empty string. For example if mint authority is disabled.

**Review by ryandgoulding — APPROVED** [review]

**Comment by nazreen** [comment]
successful initOftTx when tested with default Token Program:
https://explorer.solana.com/tx/5jjyaBjgtVrxBvhCVoknURpHMGaBkBaibFea9DoKuKVp5SmdM9t3cvaPkYx35M3cY7Kk6kzAjoXEURR38J6wiLtc?cluster=devnet

**Review by nazreen — APPROVED** [review]
